config/pmd/ruleset.xml

```diff
@@ -50,7 +55,7 @@
   </rule>
   <rule ref="category/java/design.xml/ExcessiveImports">
     <properties>
-      <property name="minimum" value="45"/><!-- should be reduced -->
+      <property name="minimum" value="50"/><!-- should be reduced -->
```

Comment:
We shouldn't really be increasing these values nor be adding extra rules

Reply:
Resolved it by restoring back to 45